proxy/proxy.go

```diff
@@ -213,6 +221,12 @@ func mapRequest(r *http.Request, rt *routing.Route, host string) (*http.Request,
 	u.Scheme = rt.Scheme
 	u.Host = rt.Host
 
+	backendURL, err := url.Parse(rt.Backend)
```

Comment:
It will not work for upgraded connections that have to get a net.Conn from the backend request.